From 2047826d868e6a6d5394cdc58b34712a36caf712 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 22 Sep 2020 12:24:25 -0700
Subject: [PATCH 1650/1852] v4l: xilinx: remapper: Fix warning for UNINIT

commit f85a5b37d5b9380806f4ec4f2ecda21f5bc230a0 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the UNINIT type.

The actual warnings are:
For xremap_set_format():
Event var_decl: Declaring variable "output" without initializer.
Event uninit_use: Using uninitialized value "output".

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-remapper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-remapper.c b/drivers/media/platform/xilinx/xilinx-remapper.c
index d2e84ec1f2d6..cfda110d1432 100644
--- a/drivers/media/platform/xilinx/xilinx-remapper.c
+++ b/drivers/media/platform/xilinx/xilinx-remapper.c
@@ -245,7 +245,7 @@ static int xremap_set_format(struct v4l2_subdev *subdev,
 			     struct v4l2_subdev_format *fmt)
 {
 	struct xremap_device *xremap = to_remap(subdev);
-	const struct xremap_mapping_output *output;
+	const struct xremap_mapping_output *output = NULL;
 	const struct xremap_mapping *mapping;
 	struct v4l2_mbus_framefmt *format;
 	unsigned int i;
-- 
2.31.1

